paraatm/tests/test_para_atm.py

```diff
@@ -94,6 +94,12 @@ def test_gate_to_gate(self):
         # Basic consistency checks:
         self.assertEqual(len(df), 369)
 
+    # Note from McFarland: testing on Ubuntu using NATS 1.8, this test
+    # often hangs after the message "Flight propagation completed",
+    # with CPU still being utilized but no further progress.  The hang
+    # occurs sometimes but other times the test completes.  This
+    # should be investigated further.  Perhaps it will be resolved by
+    # moving to GNATS.
```

Comment:
As noted in my previous note on this individual commit (0da9cf3d), you have inadvertently "undone" several of the upstream changes, one of which was to remove these comments (because the issue that they mention has been resolved).  After pulling or merging in upstream changes, you need to make sure that you are not undoing previous work when you add new commits (using `git diff` prior to making commits is the way to review all of the changes that you are about to commit).